### PR TITLE
Add Additional Permissions for Describing RIs

### PR DESCRIPTION
**Pull request by acham1**
Each AWS Organization management account or linked account should have the following actions enabled for the TernaryCMPServiceAgent role:

- redshift:DescribeReservedNodes
- es:DescribeReservedInstances
- elasticache:DescribeReservedCacheNodes
- memorydb:DescribeReservedNodes

See Jira: BIS-345, BIS-346, BIS-347, BIS-348

### CODE REVIEW

File: aws/linked-account/aws-linked-account-role-create-tform.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 locals {
-  ternary_service_account = "<INSERT_SERVICE_ACCOUNT>"
+  ternary_service_account     = "<INSERT_SERVICE_ACCOUNT>"
```
Comment (GCelichowski) [inline aws/linked-account/aws-linked-account-role-create-tform.tf:2]: Intended change?

File: aws/payer-account/aws-role-create-cfn.json [hunk 2]
```diff
@@ -121,4 +125,4 @@
             "Description": "Unique user id generated by Google."
         }
     }
-}
```
Comment (GCelichowski) [inline aws/payer-account/aws-role-create-cfn.json:124]: intended change?

File: aws/payer-account/aws-role-create-tform.tf [hunk 3]
```diff
@@ -1,6 +1,6 @@
 locals {
-  ternary_bucket_id   = "<INSERT_S3_BUCKET_NAME>"
-  ternary_service_account = "<INSERT_SERVICE_ACCOUNT>"
+  ternary_bucket_id           = "<INSERT_S3_BUCKET_NAME>"
```
Comment (GCelichowski) [inline aws/payer-account/aws-role-create-tform.tf:2]: ditto other whitespace comments above

**Comment by GCelichowski** [comment]
a couple small whitespace related changes that I assume came from a linter, but those are non-blocking, LGTM!

**Comment by acham1** [comment]
@GCelichowski thanks! I'll leave this open for a bit longer, just in case there's any last-minute addition. (The formatting changes are from `terraform fmt`)

**Comment by joshk0** [comment]
Bump, can this be merged if it's still relevant?

**Pull request merged**